File: Latihan 3/python/ram.py
class ram():

    __Capacity = ""
    __price = ""

    def __init__(self, Capacity = "", price = ""):
        self.__Capacity = Capacity
        self.__price = price

    def setCapacity(self, Capacity):
        self.__Capacity = Capacity

    def setPrice(self, price):
        self.__price = price

    def getCapacity(self):
        return self.__Capacity

    def getPrice(self):
        return self.__price

    def printRam(self):
        print("RAM Capacity : ", str(self.getCapacity()))
        print("RAM Price : ", str(self.getPrice()))

File: Latihan 3/c++/ram.cpp
#include <iostream>
#include <string>

using namespace std;

class ram{
    private:
    string capacity;
    int price;

    public:
    ram(){
        this->capacity = "";
        this->price = 0;
    }

    ram(string capacity, int price){
        this->capacity = capacity;
        this->price = price;
    }

    void setcapacity(string capacity){
        this->capacity = capacity;
    }

    void setprice(int price){
        price = price;
    }


    string getcapacity(){
        return this->capacity;
    }

    int getprice(){
        return this->price;
    }

    void printRam(){
       cout << "RAM             : " << this->capacity << endl;
       cout << "RAM Price       : " << this->price << endl;
    }

    ~ram(){}
};